site/data/docs/custom-app-info.mdx

```diff
@@ -26,3 +26,26 @@ const App = () => {
   );
 };
 ```
+
+You can also provide your dapp's Terms of Service `termsOfServiceUrl` and Disclaimer `disclaimerUrl`, which will be displayed at the bottom of the connect modal:
+
+```tsx
+import { RainbowKitProvider } from '@rainbow-me/rainbowkit';
+
+const App = () => {
+  return (
+    <RainbowKitProvider
+      appInfo={{
+        appName: 'Rainbowkit Demo',
+        termsOfServiceUrl: 'yourtermsofservice.com',
+        disclaimerUrl: 'yourtermsofservice.com',
+      }}
+      {...etc}
+    >
+      {/* ... */}
+    </RainbowKitProvider>
+  );
+};
+```
+
+Note: your Disclaimer will not be displayed without you also specifying your Terms of Service.
```

Comment:
oh this is interesting 🤔 i wonder if we can think of an API that would be more self-explanatory about this condition

right now both `termsOfServiceUrl` and `disclaimerUrl` are sibling properties and without looking at the docs there isnt much clearance that `disclaimerUrl` is dependant on `termsOfServiceUrl`

thinking out loud here, but could try and explore something like:
```
{
  appName: '',
  termsOfService: {
    url: 'https://terms-of-service.com',
    disclaimerUrl: 'https://disclaimer-url.com'
  }
}
```

any ideas?

Reply:
yes i agree, i explored this idea but determined that the above style of API is still not clear that you need one for the other

Reply:
open to ideas!

Reply:
I dont think the API above is perfect, i'm trying to think of something better. But it does mean that in order to add the `disclaimerUrl`, users must go through `termOfService` which _may_ help. 

Otherwise, if we dont think it makes much of a difference, we can consider doing a check and doing a `console.warning` or even an `Error` if `disclaimerUrl` is provided without `termsOfServiceUrl`

Last thing to consider is, where did the requirement to make disclaimer note dependant on the terms come from, and why?

Reply:
I updated this PR based on your comments @peduarte let me know what you think 🤓